### DISCUSSION TITLE
如果您的小程序项目使用了mp-cu，欢迎在这里分享。

### DISCUSSION THREAD

**Discussion by iZaiZaiA** [Color-UI/MP-CU#18] [General]
如果您的小程序项目使用了MP-CU，欢迎在这里分享您的小程序码。

我们需要一些优质的演示项目，也顺便能让更多人看见您的项目。

我们计划在仓库文档里，添加优质的小程序码展示。

### 关于如何定义优质？？

1.整体基于MP-CU组件的库设计开发。
2.漂亮、布局合理、颜色搭配合理。
3.非低俗，引流项目。
4.商业或免费都可以。

### 有时间限制吗？

没有，随时可以提交您的优质小程序码，我们会认真看完，并在团队内部进行优质打分，如果达到优质标准，就会放在仓库文档内展示。
不过，现在前期容易上榜，后期多了后，可能标准会变动。

---

我们期待能看到您的优质项目 :)

**Comment by CN-HM** [comment]
微信小程序：逃离塔克夫指南

![gh_71783db79d85_344](https://user-images.githubusercontent.com/26593955/150722226-a4618563-27a8-4805-a4dc-f55326e47a29.jpg)

**Comment by extrastu** [comment]
# 微信小程序-每图
> 故宫壁纸小程序，将历史的精彩收集到自己的手中


## 截图预览

![4611644473034_ pic_hd](https://user-images.githubusercontent.com/19451694/153347641-34d0e73e-5f80-4959-b251-465fc62d6ad2.jpg)

## 扫码体验
![gh_4d14317e25da_258](https://user-images.githubusercontent.com/19451694/153342500-f0e80ac4-e364-4eef-b94d-ffce8ca32ed6.jpg)

**Reply by qurge** [reply]
大佬，有源码吗？想学习下思路和代码设计

**Reply by caoqinen** [reply]
使用楼上开发的 小程序后  感觉沉浸式看壁纸很舒服   想请教一下

**Comment by poisonboy** [comment]
### 微慕开源小程序
WordPress生成多端小程序及APP

### 截图预览
 
![image](https://user-images.githubusercontent.com/23186549/165230490-bdf429fb-a54c-43eb-a125-274c466a5eab.png)

### 扫码体验
![image](https://user-images.githubusercontent.com/23186549/165229627-4c664e5c-efa4-4e0f-a5ac-e9b0df0a8ec8.png)


### 源码下载

[点击下载源码](https://ext.dcloud.net.cn/plugin?id=2214)

**Comment by ch4ixin** [comment]
### **解忧小工具**

解忧小工具，解不时之需，各种各样小巧便捷、十分易用的爆款小工具
视频去水印、手机清灰、识别图文、二维码生成...

###   **扫码体验**

![gh_3c0086b8f005_430](https://user-images.githubusercontent.com/35681956/188177774-3312f3a8-b8d3-4af6-bbba-d0e3691ee2c0.jpg)

**Comment by CN-HM** [comment]
商业项目 不太方便开源 有需要可以留言指导开发



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年3月23日(周三) 下午5:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Color-UI/MP-CU] 如果您的小程序项目使用了mp-cu，欢迎在这里分享。 (Discussion #18)




 
大佬，有源码吗？想学习下思路和代码设计

**Comment by extrastu** [comment]
# 口袋喇叭-地摊吆喝
> 最近家里开超市，需要生成各种吆喝，如是动手写了个小程序，欢迎试用，也希望能帮到你~

## 截图预览
![image](https://user-images.githubusercontent.com/19451694/195476730-d8fd7842-b8e9-4112-ad5a-aab342894913.png)


## 扫码体验
![image](https://user-images.githubusercontent.com/19451694/195476778-77dfee82-c48a-43e0-a867-76d79f4f4edb.png)

**Comment by Muyuteams** [comment]
微信小程序：300英雄百宝箱
![gh_28cfd5c3bf67_860](https://user-images.githubusercontent.com/121117126/208712220-6e2d2d15-c2d6-40b9-85d7-78a797602f57.jpg)

**Comment by extrastu** [comment]
# 小报童导航 - 发现优秀专栏
> 发现优秀专栏
# 截图预览
![image](https://user-images.githubusercontent.com/19451694/210199272-46c95348-8acf-435e-9e7d-ef63b4b10d4b.png)
![image](https://user-images.githubusercontent.com/19451694/210199278-44ea69bd-9cd5-4a24-896f-89702fd334d7.png)
# 扫码体验
![下载](https://user-images.githubusercontent.com/19451694/210199296-255fbd23-c0c6-4d28-b98f-df8b8f960ff0.png)

**Comment by Chadwuo** [comment]
「礼记」是一个基于微信小程序的在线人情帐簿，随身携带、快速查找、统计汇总、亲友往来、家庭共享记账，为记账带来了全新体验。
---
### 在线体验

![](https://github.com/Color-UI/MP-CU/assets/41413559/3055bfd6-0a5c-4e26-9ad7-3820b4a9f4ec)

### 效果图

![](https://github.com/Color-UI/MP-CU/assets/41413559/2cf018e2-772d-42d6-b355-2fdf55f9fc03)

**Comment by qywk99** [comment]
# 玩转线性代数
一款辅助学生学习的线性代数小程序，帮助初学者解决计算的痛点，提高学习效率。
![玩转线性代数小程序码](https://user-images.githubusercontent.com/45910487/222476471-7ed978eb-5731-4a1b-a1f2-a0a19a18b7d0.jpg)
